src/rota/govuk_2ndline.py

```diff
@@ -96,6 +103,28 @@ def generate_model(
                         pulp.lpSum(rota[week, person, role.name] for role in Roles if role.value.inhours) + pulp.lpSum(rota[week - 1, person2, role.name] for role in Roles if role.value.inhours) <= 1
                     )
 
+        # [2.7] Not to be assigned more than max_inhours_standby_shifts_per_person in-hours roles in total
+        prob += pulp.lpSum(rota[week, person, role.name] for week in range(num_weeks) for role in Roles if role.value.inhours_standby) <= max_inhours_shifts_per_person
```

Comment:
Should be `<= max_inhours_standby_shifts_per_person`

Reply:
Thanks @barrucadu ! I've updated it [here](https://github.com/barrucadu/govuk-rota-generators/pull/15/files#diff-62cdfe7af97a0652a2465474a89a5b0c23c80d11bf0dc47b49a0b0767562f9f6R118)